artemis/reporting/modules/mail_dns_scanner/reporter.py

```diff
@@ -74,6 +76,22 @@ def create_reports(task_result: Dict[str, Any], language: Language) -> List[Repo
             )
         return result
 
+    @staticmethod
+    def get_alerts(all_reports: List[Report], reports_for_top_level_target_threshold: int = 8) -> List[str]:
```

Comment:
104 chars long, but from what I can see, artemis is fine with 120char line length.